lib/transport/data_validation/commands/create_project.ex

```diff
@@ -9,11 +9,6 @@ defmodule Transport.DataValidation.Commands.CreateProject do
       ...> |> CreateProject.validate
       {:ok, %CreateProject{name: "transport"}}
 
-      iex> %{name: "covoiturage", autoFetchFeeds: "true"}
-      ...> |> CreateProject.new
-      ...> |> CreateProject.validate
-      {:ok, %CreateProject{name: "covoiturage"}}
-
```

Comment:
Why did you remove that?

Reply:
I thought it was over-specification, though I can add it back.